src/assertDefaultState.js

```diff
@@ -0,0 +1,9 @@
+import isUndefined from 'lodash/isUndefined';
+
+export default function assertDefaultState(defaultState, actionTypes) {
+  if (isUndefined(defaultState)) {
+    throw new Error(
+      `Expected defaultState for reducer handling ${actionTypes.join(', ')} to be defined`
+    );
+  }
+}
```

Comment:
I'd suggest to move this function to a folder named `utils` and name that file `assertions.js` or sth like this, where we can write more functions like this in a single file.